compiler/rustc_codegen_gcc/tests/run/abort1.rs

```diff
@@ -36,10 +36,7 @@ mod intrinsics {
 
     #[rustc_nounwind]
     #[rustc_intrinsic]
-    #[rustc_intrinsic_must_be_overridden]
-    pub fn abort() -> ! {
-        loop {}
-    }
+    pub fn abort() -> !;
```

Comment:
Oh, it seems I forgot to remove and use the one from `mini_core`. I created [an issue](https://github.com/rust-lang/rustc_codegen_gcc/issues/637) for this.